src/test/java/uk/nhs/hee/tis/trainee/sync/service/PlacementSyncServiceTest.java

```diff
@@ -114,7 +123,9 @@ void shouldNotFindRecordByIdPostWhenNotExists() {
   }
 
   @Test
-  void shouldSendRetrievalRequest() {
-    assertThrows(UnsupportedOperationException.class, () -> service.request(ID));
+  void shouldSendARequestForPlacementToTisSync() throws JsonProcessingException {
```

Comment:
Not a major issue, but the naming suggests we're able to test where the message goes. We only verify that a request is sent.